database_manager/command_handlers/helpers/database_updater.py

```diff
@@ -0,0 +1,123 @@
+from typing import Union
+from sqlalchemy.orm.query import Query
+
+from database_manager.database_tables.groups_and_users import Users
+from database_manager.database_tables.mustwatches import UserRequests, Watches, Mustwatches
+
+from telebot_controller.command_handlers.helpers.buttons import Button as btn
+
+
+class DatabaseUpdater:
```

Comment:
I see you are splitted your services on Updater and Checker. Please split them on domain areas instead.